benchmark/src/main/scala/keycloak/scenario/CommonSimulation.scala

```diff
@@ -36,7 +38,12 @@ abstract class CommonSimulation extends Simulation {
   }
 
   def setUp(name: String, builder : KeycloakScenarioBuilder): Unit = {
-    val scn = scenario(name).exec(builder.chainBuilder)
+    // Add an attribute which uniquely identifies the Gatling user
+    val gatlingUniqueUserId = new AtomicLong(0)
```

Comment:
To be able to run Gatling multiple times against the same DB, an ID that would be unique across multiple runs would be helpful.

On another note: if the realm IDs generated here would actually be UUIDs there would be no need for such a state, and even multiple Gatling instances could run in parallel. 

```suggestion
    // Assuming less than 1000 users per second, this will be unique across multiple runs.
    val gatlingUniqueUserId = new AtomicLong(System.currentTimeMillis())
```

Reply:
I considered using a UUID instead, but was feeling it might get difficult to handle if investigating, especially with the UI.
However, I do believe that's better to use a UUID - it gracefully handles the case where the scenario is run multiple times against the same deployment (currently, any subsequent run will fail because will attempt to create realms with already existing id, leading to a conflict).

Do you think I should use the already existing UUID named `state` ?
Not sure about the semantics of this one, so it might be better to add a new UUID for this specific purpose.

Reply:
I had a look and I see that the UUID `state` is used for the OpenID connect login. Therefore that wouldn't fit here, IMHO. 

I suggest you'd add a UUID as part of `createRealm()` (similar to your latest change of "requestEpoch"), although this should then be named specific to the this step like 
`createdRealmId`, so that it can later be used to delete that realm.

Reply:
Ok, makes sense. Will do.